packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -237,51 +245,39 @@ class RenderableTiledMap {
 
 abstract class _RenderableLayer<T extends Layer> {
   final T layer;
+  final Vector2 destTileSize;
+  final TiledMap map;
```

Comment:
Is it possible to not have these in here? It look like it is unnecessarily increasing all the derived classes as well. For example, `_UnrenderableLayer` is not even using them.

Reply:
Yes, I think it's possible. Let work on it.

Reply:
Actually; they are needed for the scale and offset calculations.  Tile, Image, and Group uses these. `_UnrenderableLayer` is just unlucky.  `RenderableTiledMap` is allocated after the layers are created... maybe we just update the layers to point to `RenderableTiledMap` as the parent map?

Reply:
Got idea. Also, unrenderable layer is basically the object layer and technically it can contain renderable tile objects. So this change is actually useful for adding rendering support for the object layer in future.